site/src/examples/card/DisabledInteractableCard.tsx

```diff
@@ -0,0 +1,12 @@
+import { ReactElement } from "react";
+import { InteractableCard } from "@salt-ds/core";
+
+export const DisabledInteractableCard = (): ReactElement => {
+  return (
+    <InteractableCard
+      accent="top"
+      style={{ width: "260px", height: "144px" }}
```

Comment:
I saw the kickoff says there will be S/M/L sizes rather than width/height - is that no longer the case or still needs to be implemented?

Reply:
ah yeah the sizes was referring to padding, which we now just have an example to show how to override the style. Height/width of card just fills container